snaps/reference/cli/subcommands.md

```diff
@@ -9,160 +9,176 @@ import TabItem from '@theme/TabItem';
+
+### `c`, `config`
 
 <Tabs>
 <TabItem value="Syntax">
 
 ```bash
-yarn mm-snap build [options]
+yarn mm-snap build --config <FILE>
 ```
 
 </TabItem>
 <TabItem value="Example">
 
 ```bash
-yarn mm-snap b -s lib/index.js -d out -n snap.js
+yarn mm-snap build --config ./snap.config.build.ts
 ```
 
 </TabItem>
 </Tabs>
 
-Builds a Snap from source.
+Path to the [configuration file](../../learn/about-snaps/files.md#configuration-file).
 
-`b` is an alias for `build`.
+`-c` is an alias for `--config`.
 
-## e, eval
-
-<Tabs>
-<TabItem value="Syntax">
+## `e`, `eval`
 
 ```bash
-yarn mm-snap eval [options]
+yarn mm-snap eval
 ```
 
-</TabItem>
-<TabItem value="Example">
+Attempts to evaluate the Snap bundle in
+[Secure ECMAScript (SES)](../../learn/about-snaps/execution-environment.md#secure-ecmascript-ses).
+
+`e` is an alias for `eval`.
+
+## `m`, `manifest`
 
 ```bash
-yarn mm-snap e -b out/snap.js
+yarn mm-snap manifest
 ```
 
-</TabItem>
-</Tabs>
-
-Attempts to evaluate the Snap bundle in SES.
+Validates the Snap [manifest file](../../learn/about-snaps/files.md#manifest-file).
 
-`e` is an alias for `eval`.
+`m` is an alias for `manifest`.
 
-## i, init
+### `fix`
 
 <Tabs>
 <TabItem value="Syntax">
 
 ```bash
-yarn mm-snap init [directory]
+yarn mm-snap manifest --fix <BOOLEAN>
 ```
 
 </TabItem>
 <TabItem value="Example">
 
 ```bash
-yarn mm-snap i my-snap
+yarn mm-snap manifest --fix false
 ```
 
 </TabItem>
 </Tabs>
 
-Initializes a Snap project in the specified directory.
-If no directory is specified, the Snap project is initialized in the current directory.
+Enables or disables making any changes to fix the manifest file.
+The default is `true`.
+
+## `s`, `serve`
+
+```bash
+yarn mm-snap serve
+```
+
+Locally serves Snap files for testing.
 
-`i` is an alias for `init`.
+`s` is an alias for `serve`.
 
-## m, manifest
+### `p`, `port`
 
 <Tabs>
 <TabItem value="Syntax">
 
 ```bash
-yarn mm-snap manifest [options]
+yarn mm-snap serve --port <PORT>
 ```
 
 </TabItem>
 <TabItem value="Example">
 
 ```bash
-yarn mm-snap m --fix false
+yarn mm-snap serve --port 9000
 ```
 
 </TabItem>
 </Tabs>
 
-Validates the Snap [manifest file](../../learn/about-snaps/files.md#manifest-file).
+The local server port for testing.
+The default is `8081`.
 
-`m` is an alias for `manifest`.
+`-p` is an alias for `--port`.
+
+## `w`, `watch`
 
-## s, serve
+```bash
+yarn mm-snap watch
+```
+
+Rebuilds a Snap from source upon changes.
+
+`w` is an alias for `watch`.
+
+### `p`, `port`
 
 <Tabs>
 <TabItem value="Syntax">
 
 ```bash
-yarn mm-snap serve [options]
+yarn mm-snap watch --port <PORT>
 ```
 
 </TabItem>
 <TabItem value="Example">
 
 ```bash
-yarn mm-snap s -r out -p 9000
+yarn mm-snap watch --port 9000
 ```
 
 </TabItem>
 </Tabs>
 
-Locally serves Snap files for testing.
+The local server port for testing.
+The default is `8081`.
 
-`s` is an alias for `serve`.
+`-p` is an alias for `--port`.
 
-## w, watch
+## Global options
```

Comment:
Heading looks weird in published output

Reply:
I think we'll fix this in a separate issue #1010